scipy/special/tests/test_pdtr.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
NumPy is a third party import, so by PEP8 it should be in the same cluster as the `pytest` one.